### PR TITLE
#167165077 Users should be able to like a specific comment

### PR DESCRIPTION
**Pull request by tolulope-od**
#### What does this PR do?

Adds functionality to like a single comment
Changes comment model to include likes
Changes the like model to include like boolean values

#### Description of Task to be completed?

Users should be able to like a specific comment by hitting the route `POST -> /api/v1/comments/likes/:commentId`

*Flowchart for API flow to like a comment*
Like a Comment API Flowchart (3)

#### How should this be manually tested?

After cloning this repo, `cd` into it and do the following:

```bash
# fetch this branch
$ git fetch origin  ft-like-a-comment-167165077

# install dependencies
$ npm install

# run unit tests
$ npm test

# start the development server
$ npm run server
```
Using postman, log in or register an account and hit the post route to create an article (user must be verified via the verification route in #28 
After getting the token, add it to the header and make a request to the route stated above

#### Any background context you want to provide?
N/a

#### What are the relevant pivotal tracker stories?
[#167165077](https://www.pivotaltracker.com/story/show/167165077)

#### Screenshots (if appropriate)

*User successfully liked a comment*
Screenshot 2019-08-06 at 11 53 04

*User removed their like for an article*
Screenshot 2019-08-06 at 11 56 21

### CODE REVIEW

File: controllers/CommentController.js [hunk 1]
```diff
@@ -42,4 +44,80 @@ export default class CommentController {
...
+            likes: foundComment.dataValues.likes - 1,
+          },
+          {
+            where: {
+              id: foundComment.dataValues.id,
+            }
+          }
+        );
+
+        return successResponse(res, 200, 'comment', { message: 'Removed your \'like\' for this article' });
```
Comment (OdunayoOkebunmi) [inline controllers/CommentController.js]: Can you use a better response? For instance " 'like' has been removed"

**Review by OdunayoOkebunmi — APPROVED** [review]

**Review by PelumiAlesh — APPROVED** [review]
Looks good to me

**Review by Tunji17 — APPROVED** [review]
LGTM

**Review by mekzy-o — APPROVED** [review]
Nice Job Tolu. looks good to me

**Pull request merged**